src/20250319_Multi_FinanceStewardDeploymentPoolExposureModule/FinanceStewardDeploymentPoolExposureModule_20250319.s.sol

```diff
@@ -0,0 +1,338 @@
+      memory actionsEthereum = new IPayloadsControllerCore.ExecutionAction[](2);
+    actionsEthereum[0] = GovV3Helpers.buildAction(
+      type(AaveV3Ethereum_FinanceStewardDeploymentPoolExposureModule_20250319).creationCode
+    );
+    actionsEthereum[1] = GovV3Helpers.buildAction(
+      type(AaveV3EthereumLido_FinanceStewardDeploymentPoolExposureModule_20250319).creationCode
+    );
+    payloads[0] = GovV3Helpers.buildMainnetPayload(vm, actionsEthereum);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsPolygon = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsPolygon[0] = GovV3Helpers.buildAction(
+      type(AaveV3Polygon_FinanceStewardDeploymentPoolExposureModule_20250319).creationCode
+    );
+    payloads[1] = GovV3Helpers.buildPolygonPayload(vm, actionsPolygon);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsAvalanche = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsAvalanche[0] = GovV3Helpers.buildAction(
+      type(AaveV3Avalanche_FinanceStewardDeploymentPoolExposureModule_20250319).creationCode
+    );
+    payloads[2] = GovV3Helpers.buildAvalanchePayload(vm, actionsAvalanche);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsOptimism = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsOptimism[0] = GovV3Helpers.buildAction(
+      type(AaveV3Optimism_FinanceStewardDeploymentPoolExposureModule_20250319).creationCode
+    );
+    payloads[3] = GovV3Helpers.buildOptimismPayload(vm, actionsOptimism);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsArbitrum = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsArbitrum[0] = GovV3Helpers.buildAction(
+      type(AaveV3Arbitrum_FinanceStewardDeploymentPoolExposureModule_20250319).creationCode
+    );
+    payloads[4] = GovV3Helpers.buildArbitrumPayload(vm, actionsArbitrum);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsBase = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsBase[0] = GovV3Helpers.buildAction(
+      type(AaveV3Base_FinanceStewardDeploymentPoolExposureModule_20250319).creationCode
+    );
+    payloads[5] = GovV3Helpers.buildBasePayload(vm, actionsBase);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsScroll = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsScroll[0] = GovV3Helpers.buildAction(
+      type(AaveV3Scroll_FinanceStewardDeploymentPoolExposureModule_20250319).creationCode
+    );
+    payloads[6] = GovV3Helpers.buildScrollPayload(vm, actionsScroll);
+
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actionsBNB = new IPayloadsControllerCore.ExecutionAction[](1);
+    actionsBNB[0] = GovV3Helpers.buildAction(
+      type(AaveV3BNB_FinanceStewardDeploymentPoolExposureModule_20250319).creationCode
+    );
+    payloads[7] = GovV3Helpers.buildBNBPayload(vm, actionsBNB);
+
+    IPayloadsControllerCore.ExecutionAction[]
```

Comment:
For linea there are currently issues in the create2 prediction as the script will run with shanghai, but the payload is deployed with london.

To work around, you can follow the approach taken here:
https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20250228_Multi_ClinicStewardActivation/ClinicStewardActivation_20250228.s.sol#L387-L394
1. deploy and register the payload
2. add the payload id here, instead of inferring it from the address